libs/eo/help/shell/src/lib/eo-geography-page.component.ts

```diff
@@ -15,12 +15,174 @@
+        background-color: var(--watt-color-primary-light);
+
+        p {
+          padding: var(--watt-space-m);
+
+          b {
+            color: var(--watt-color-primary-dark);
+          }
+        }
+      }
+
+      .info-box {
+        padding: var(--watt-space-m);
+
+        &.green {
+          background-color: var(--watt-color-primary-dark);
+          color: var(--watt-color-neutral-white);
+
+          span {
+            color: var(--watt-color-neutral-white);
+          }
+        }
+
+        &.beige {
+          background-color: var(--watt-color-secondary-light);
+        }
+
+        img {
+          display: block;
+          padding-bottom: var(--watt-space-m);
+        }
+      }
+    `,
+  ],
+  template: `
+    <div>
+      <h3>Strøm kan ikke flyttes på tværs af hele Europa</h3>
+
+      <p>
+        Det er ikke uden udfordringer, når energi flyttes gennem elnettet. Det
+        gælder både, når der er tale om at flytte energi indenfor egne
+        landegrænser, men særligt når det er på tværs af landegrænser. Ved
+        dokumentation og sporing af energis oprindelse, for at sikre at energien
+        er grøn, er det vigtigt, at der tages højde for disse udfordringer.
+      </p>
+
+      <h4>Grøn produktion kan være sort forbrug</h4>
+      <p>
+        En af problematikkerne findes bl.a. ved køb af GO certifikater. Det er
+        f.eks. lige nu muligt for danske virksomheder at købe GO certifikater af
+        grøn energi produceret i Island. På papiret ser det på den måde ud som
+        om, at den energi man forbruger er grøn. Virkeligheden kan dog være en
+        helt anden. Da der ikke findes nogen forbindelse mellem Island og
+        Danmark, så energien produceret i Island kan umuligt være samme energi
+        forbrugt i Danmark.
+      </p>
+
+      <figure>
+        <img
+          width="367"
+          src="/assets/images/help/map_geography.jpg"
+          alt="Transmission net across countries"
+        />
+        <figcaption>
+          Billedet viser det europæiske transmissionsnet på tværs af
+          landegrænser
+        </figcaption>
+      </figure>
+
+      <h3>Hvordan løser vi udfordringerne?</h3>
+      <p>
+        En løsning er at sætte regler for, hvor lang afstanden må være fra
+        produktions- til forbrugsstedet ved handel af grønne certifikater.
+        Begrænsningerne sættes af de fysiske kabler, altså at man kun kan handle
+        certifikater fra nabolande. På den måde sikrer man, at grøn energi kun
+        kan bruges i produktionslandet og eventuelle nabolande.
+      </p>
+      <div class="case">
+        <img
+          alt="Geography case"
+          src="/assets/images/help/case_geography.jpg"
+        />
+        <p>
+          <b>CASE</b><br />
+          Peter Producent kan sælge sine certifikater i Danmark, Tyskland, Norge
+          og Sverige. Så længe der er plads på de respektive forbindelser.
+        </p>
+      </div>
+      <h4>
```

Comment:
Not really a header. Likely just used for styling?

Reply:
Removed header :) Good catch